port_ocean/core/defaults/initialize.py

```diff
@@ -73,7 +73,8 @@ async def _initialize_required_integration_settings(
                 port_app_config=default_mapping,
             )
     except httpx.HTTPStatusError as err:
-        raise Exception(f"Failed to apply default mapping: {err.response.text}.")
+        logger.error("Failed to apply default mapping")
```

Comment:
```suggestion
        logger.error(f"Failed to apply default mapping: {err.response.text}.")
```

Reply:
that was the previous state, don't think it's needed to both log the error text and raise the whole exception

Reply:
The whole exception wont log the response text